### ISSUE TITLE
Developing Processes/Best Practices for program area

### ISSUE THREAD

**Issue by ayushdamani** [bcgov/digital-journeys#1211]
User Story
As a program area, we need to have a clear process and approach in place before we build, so that we can deliver consistent expert products and experiences for our clients and create a predictable reliable work practice for our team

this could include:

- requirements for business area
- translated into technical requirements
- Include architectural drawings before starting development
(How formsflow.ai is technically structured AND how a project looks after the UX sessions - help inform the build)

Tasks

- [ ] explore - need to look into options
- [ ] Steps needed to achieve this processes
- [ ] break this down into smaller tasks project for each piece of collateral

Acceptance Criteria
Team members know what the next steps in any project are, can easily identify when things aren't going to plan and bring project pack on track.
knowledge transfer processes are streamlined
Clients experience is improved through clearing knowing what to expect form the Digital Journeys Experience

**Comment by Stella-Archer** [comment]
New process developing project by doing application flow diagrams 1st

**Comment by Stella-Archer** [comment]
try to diagram pre-adoption from screen to BPMN diagram - new ticket

**Comment by Stella-Archer** [comment]
1/ high level  seven step what goes on
2/ process logic on how it flows (business rules and functions) mid level
3/ super detail numbered of what's happening in the micros steps and options
